package.json

```diff
@@ -1703,7 +1703,7 @@
     "tslint": "^5.20.1",
     "typedoc": "^0.20.25",
     "typescript": "^4.1.5",
-    "vscode": "^1.1.36",
+    "vscode": "^1.53.0",
     "vscode-nls-dev": "^3.3.2",
     "webpack": "^5.22.0",
```

Comment:
These version changes are causing CI to fail. No idea why but lets' experiment with versions and see if we can unblock the build.

Reply:
There shouldn't be a need to update the version so high.  I'm not sure when this API went in, but you don't need latest.

I believe "vscode" doesn't go above a certain number.  They changed the way they distribute the API around 1.40 or so, I think.  cpptools uses this now: 
```
   "@types/vscode": "1.52.0",
```
...but I think there's more to it than just making this small change.  You might want to sync with @Colengms since he made that change a while back.

Reply:
It is changed back to older version